css/layout.css

```diff
@@ -1018,20 +1018,20 @@ section.communityBox .section-header{
 #up{
      display: none;
      position: fixed;
-     bottom: 10vh;
-     right: 5vw;
+     bottom: 4vh;
+     right: 2vw;
      height: 50px;
      width: 50px;
      border-radius: 25px;
      z-index: 10;
      cursor: pointer;
-     background-color: #45006e;
+     background-color: #1a8cff;
      padding: 15px;
      box-shadow: 0 0 5px 0 #ccccff;
 	 transition: all 0.2s linear;
 }
 #up:hover,#up:active{
-     background-color: #620090;
+     background-color: #1a8cff;

```

Comment:
Make this color of a bit different shade (preferably a darker one) to give it a visible hover/touch effect.

Reply:
Do you mean a darker blue or a darker color in general ?

Reply:
A darker blue.

Reply:
![screenshot from 2018-04-02 07-43-50](https://user-images.githubusercontent.com/35984969/38199393-4f694ff6-36ae-11e8-818b-07a84387a34d.png)
a little darker,subtle difference tho